src/config.rs

```diff
@@ -335,6 +339,10 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
         xds_root_cert,
         ca_address,
         ca_root_cert,
+        secret_ttl: match parse::<i64>(SECRET_TTL)? {
```

Comment:
Why don't you use parse_default

Reply:
can it parse string like "24h"

Reply:
I believe this function cannot parse time duration like `1h` `1m`

Reply:
[duration-str](https://docs.rs/duration-str/latest/duration_str/) looks like it works.  Now accepts times in the form of `5m`, `1h` etc

Reply:
Looks good